tests/ext-popup.test.js

```diff
@@ -23,187 +26,205 @@ function observeChange(element) {
   });
 }
 
-test('start monitoring all extensions from the popup', async () => {
+const getMonitorStatusSpyFn = jest.spyOn(ExtListen, 'getMonitorStatus');
+const startMonitorSpyFn = jest.spyOn(ExtListen, 'startMonitor');
+const stopMonitorSpyFn = jest.spyOn(ExtListen, 'stopMonitor');
+
+afterEach(() => {
+  getMonitorStatusSpyFn.mockReset();
+  startMonitorSpyFn.mockReset();
+  stopMonitorSpyFn.mockReset();
+});
+
+test('clicking on start monitoring button should display that the extensions are being monitored', async () => {
```

Comment:
This description - `clicking on start monitoring button should display that the extensions are being monitored` is more aligned with what a user should expect after clicking the start monitoring button rather than a description like - `start monitoring all extensions from the popup`. So, I believe a new developer will find the new test name more easily understandable.